OrangeDesignSystemDemo/OrangeDesignSystemDemo/Resources/Base.lproj/Localizable.strings

```diff
@@ -35,13 +35,14 @@
 "shared.trailing" = "Trailing";
 "shared.recipes" = "Recipes";
 "shared.foods" = "Foods";
+"shared.loading" = "Loading\U2026";
 
 // =====================
 // MARK: - Miscellaneous
 // =====================
 
 "misc.usage" = "Usage";
-"misc.coming_soon" = "Coming soon...";
+"misc.coming_soon" = "Coming soon\U2026";
```

Comment:
why ?

Reply:
Ellipsis symbols should be used for accessibility reasons. In fact "..." is a whole symbol and should not be replaced by three dots "." :)